modules/ROOT/pages/policies-policy-overview.adoc

```diff
@@ -0,0 +1,87 @@
+
+However, the Anypoint Platform agent remains connected to API Manager. After policies are reconfigured or removed from API Manager, those policies are downloaded to any connected API Gateway or Mule runtimes, which updates each runtime /policies folder. The policy changes are then again injected into each API proxy application. This allows policies to be dynamically changed without having to redeploy the API proxy application, and without having to restart the API Gateway or Mule runtimes.
+
+
+
+== Policy Scope
+
+Policies are isolated from the application and other policies. However, there are ways to expose information from a policy. The authentication information about a policy can be propagated in the Security Context Principal object.
+
+Some policies, such as the token enforcement policy, allow propagation of headers, including the information related to the token. This token is used by the implementation service or by the API itself.
+
+//Eva--add info about the generic policy scope Add an example or link here..Fede Balbi?
+
+
+== Policy Size
+
+The size of a policy varies based on its dependencies. The first time Mule Runtime polls for policies it will take longer to fetch them. However, after policies are retrieved, they are cached on the file system. This ensures offline availability and also reduces latency to fetch policies.
+//Add more information about the policy size. we have two options here, either hide the policy size, or clarify that this doesn't happen for cloudhub or runtime fabric--chirag
+
+
+
+== Policy Level
+
+Policies implement an additional level of granularity, where access is controlled based on a criteria. In this level of granularity, policies are applied to only those request that match the criteria.  These polices are known as resource-level policies. All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
+
+Resource level policy support includes RAML and HTTP-based APIs (since Mule 4). A policy applied at a resource level affects all HTTP methods, for example, PUT, GET, POST, or selected HTTP methods, within the resource.
+
+image::methods-resources.png[]
+
+You can apply multiple conditions to filter your resources and HTTP methods. The URI template regex can be applied to one, many, or all the methods that your API has. The uses for resource level policies are limitless. Listed below are a few examples of how you might implement the resource-level policies:
+
+* Applying policies to specific resources
+* Securing a subset of an API
+* Setting different limits on resources
+
+If you have a RAML/OAS spec attached to your API, click Preview Resource Matching to check which resources your filters affect.
```

Comment:
```suggestion
If you have a RAML or OAS spec attached to your API, click *Preview Resource Matching* to check which resources your filters affect.
```
I have edited all of this text before; does it appear more than once in this PR? I had comments earlier about WHERE the user is clicking this control.

Reply:
Done! Thanks! I had to remove it from the overview, which I did now.